Source/ACE/Entity/Door.cs

```diff
@@ -104,22 +105,25 @@ public float ResetInterval
             private set;
         }
 
+        private double? resetTimestamp;
         public double? ResetTimestamp
         {
-            get;
-            private set;
+            get { return resetTimestamp; }
```

Comment:
Why are these values public?  Reading them is probably racy, so its dangerous making them public without explicit purpose.